aws-appsync/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=appsync
+POM_NAME=Amplify Framework for Android - AppSync
+POM_DESCRIPTION=Amplify Framework for Android - AppSync
```

Comment:
We can likely be more descriptive here. We should discuss its usage - shared components used by x, y categories for interacting with AWS AppSync.

Reply:
updated